src/test/java/com/easypost/BillingTest.java

```diff
@@ -1,46 +1,25 @@
 package com.easypost;
 
 import com.easypost.exception.EasyPostException;
-import com.easypost.http.Requestor;
 import com.easypost.http.Requestor.RequestMethod;
+import com.easypost.mocking.MockClient;
+import com.easypost.mocking.MockRequest;
+import com.easypost.mocking.MockRequestMatchRules;
+import com.easypost.mocking.MockResponse;
 import com.easypost.model.PaymentMethod;
 import com.easypost.model.PaymentMethodObject;
-import org.junit.jupiter.api.AfterAll;
-import org.junit.jupiter.api.BeforeAll;
 import org.junit.jupiter.api.Test;
-import org.mockito.MockedStatic;
-import org.mockito.Mockito;
 
 import static org.junit.jupiter.api.Assertions.assertDoesNotThrow;
 import static org.junit.jupiter.api.Assertions.assertNotNull;
 
 public final class BillingTest {
-    private static TestUtils.VCR vcr;
-    private String jsonResponse = "{\"id\":\"cust_...\",\"object\":\"PaymentMethods\",\"primary_" +
-            "payment_method\":{\"id\":\"card_...\",\"disabled_at\":null,\"object\":\"CreditCard\",\"na" +
-            "me\":null,\"last4\":\"4242\",\"exp_month\":1,\"exp_year\":2025,\"brand\":\"Visa\"},\"secondar" +
-            "y_payment_method\":{\"id\":\"card_...\",\"disabled_at\":null,\"object\":\"CreditCard\",\"name\":nu" +
-            "ll,\"last4\":\"4444\",\"exp_month\":1,\"exp_year\":2025,\"brand\":\"Mastercard\"}}";
-    private PaymentMethod paymentMethod = Constants.Http.GSON.fromJson(jsonResponse, PaymentMethod.class);
-    private static MockedStatic<Requestor> requestMock = Mockito.mockStatic(Requestor.class);
-
-    /**
-     * Setup the testing environment for this file.
-     *
-     * @throws EasyPostException when the request fails.
-     */
-    @BeforeAll
-    public static void setup() throws EasyPostException {
-        vcr = new TestUtils.VCR("billing", TestUtils.ApiKey.PRODUCTION);
-    }
-
-    /**
-     * Release the static mock once it has been used.
-     */
-    @AfterAll
-    public static void cleanup() {
-        requestMock.close();
-    }
+    private final String retrievePaymentMethodResponseJson =
+            "{\"id\":\"cust_...\",\"object\":\"PaymentMethods\",\"primary_" +
+                    "payment_method\":{\"id\":\"card_...\",\"disabled_at\":null,\"object\":\"CreditCard\",\"na" +
+                    "me\":null,\"last4\":\"4242\",\"exp_month\":1,\"exp_year\":2025,\"brand\":\"Visa\"},\"secondary_" +
+                    "payment_method\":{\"id\":\"card_...\",\"disabled_at\":null,\"object\":\"CreditCard\",\"name\":nu" +
+                    "ll,\"last4\":\"4444\",\"exp_month\":1,\"exp_year\":2025,\"brand\":\"Mastercard\"}}";
```

Comment:
Can we format this string to something like [this](https://github.com/EasyPost/easypost-java/blob/master/src/test/java/com/easypost/ErrorTest.java#L176-%20L195) that I have previously done for error test?